.env.example

```diff
@@ -8,6 +8,9 @@ WAVELINK_NODE_PASSWORD='password'
 SPOTIFY_CLIENT_ID='client_id'
 SPOTIFY_CLIENT_SECRET='client_secret'
 
+OUTRO_VIDEO_URL='youtube_url'
+OUTRO_VIDEO_TIMESTAMP=12345 #timestamp in milliseconds
```

Comment:
rename to `OUTRO_VIDEO_TIMESTAMP_MS`, it's generally a good practice to include units in variables' names and you'll avoid the comment `#timestamp in milliseconds`